Updated README.md with additional details and fixed outdated script/co… (#104)
AutoResearch/autora: README.md

```shell
pyenv init
``` 
... and follow the instructions to add `pyenv` to the `$PATH` by editing the interactive shell configuration 

> file, `.zshrc`. Typically, this file is a hidden file in your home directory. You can create or edit this file using a 

Reviewer: ```suggestion
file, `.zshrc` or `.bashrc`. If it exists, this file is a hidden file in your home directory. You can create or edit this file using a 
```

> file, `.zshrc` or `.bashrc`. If it exists, this file is a hidden file in your home directory. You can create or edit this file using a 

Reviewer: The "Missing Semester of your CS Education" has a good article on dotfiles for the interested. Shall we include a link to that?
```suggestion
file, `.zshrc` or `.bashrc`. If it exists, this file is a hidden file ([dotfile](https://missing.csail.mit.edu/2019/dotfiles/)) in your home directory. You can create or edit this file using a 
```

> text editor or with CLI commands. Add the lines of script from the `pyenv init`response to the `.zshrc` file if they are 

Reviewer: ```suggestion
text editor or with CLI commands. Add the lines of script from the `pyenv init` response to the `.zshrc` file if they are 
```
